content/tutorials/grafana-fundamentals.md

```diff
@@ -106,216 +119,234 @@ To the far left you can see the _sidebar_, a set of quick access icons for navig
+As you can see in the graph legend, the query returns a time series for every method, route, and status code. ((Wait, what? No it doesn't. This is confusing. You need to tell them what the legend is on the screen (different colored lines) and explain this more simply. At first, I was confused because job and instance were all listed, but after inspecting them for a few seconds, I realized that those were all the same.)) 
 
-- Add the `sum` function to your query to group time series by route:
+- PromQL lets you group time series by labels. Add the `sum` function to your query to group time series by route:
 
 ```
 sum(rate(tns_request_duration_seconds_count[5m])) by(route)
 ```
 
-- Go back to the sample application and generate some traffic by adding new links, voting, or just refresh the browser.
+- Go back to the sample application and generate some traffic by adding new links, voting, or just refresh the browser. ((This would be a great place to have them change the time picker at the top. I added some stuff, but I thought nothing changed because it was just very small lines at the far right. If they change the time picker to 5 minutes or 15 minutes, then the changes will be far more obvious.))
 
-> Depending on your use case, you might want to group on other labels. Try grouping by other labels, such as `status_code`, by changing the `by(route)` part of the query.
+Depending on your use case, you might want to group on other labels. Try grouping by other labels, such as `status_code`, by changing the `by(route)` part of the query.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Add a logging data source" %}}
 
-Grafana 6.0 introduced support for logging data sources, like [Loki](https://grafana.com/oss/loki/). Just like for metrics, you first need to add your data source to Grafana.
+Grafana supports log data sources, like [Loki](https://grafana.com/oss/loki/). Just like for metrics, you first need to add your data source to Grafana.
 
-- In the side bar, click **Configuration** -> **Data Sources**.
-- Click **Add data source**, and select "Loki" from the list of available data sources.
-- In the URL box, type [http://loki:3100](http://loki:3100).
+- In the side bar, hover your cursor over the  **Configuration** (gear) icon, and then click **Data Sources**.
+- Click **Add data source**. 
+- In the list of data sources, click **Loki**.
+- In the URL box, enter [http://loki:3100](http://loki:3100).
 - Click **Save & Test** to save your changes.
 
 Loki is now available as a data source in Grafana.
 
 {{% /tutorials/step %}}
 {{% tutorials/step title="Explore your logs" %}}
 
-Grafana Explore not only lets you make ad-hoc queries for metrics, but lets you explore your logs as well.
+Grafana Explore not only lets you make ad-hoc queries for metrics, but lets you explore your logs as well. ((MARCUS - Remember, I am a noob. I don't know what an ad hoc query is, so you should explain it.))
```

Comment:
Good point. Maybe we can use another word instead. "Interactive metrics"?

Reply:
Or define "ad hoc query," because I hear people say it a lot.